griddb_fdw.c

```diff
@@ -950,6 +955,8 @@ griddbIterateForeignScan(ForeignScanState *node)
 static void
 griddbReScanForeignScan(ForeignScanState *node)
 {
+	elog(DEBUG1, "%s", __func__);
```

Comment:
If we should this debug log stay in the code, please follow the same format with existing log in griddbEndForeignInsert.

Reply:
I revised elog function for debug.
I added elog(DEBUG1,...) in all the callback routines.